src/opm/input/eclipse/EclipseState/InitConfig/InitConfig.cpp

```diff
@@ -120,6 +127,17 @@ namespace Opm {
 
         return this->equil;
     }
+    
+    bool InitConfig::hasStressEquil() const {
+        return !this->equil.empty();
```

Comment:
Bug: should check stressequil, not equil.

Reply:
fixed